agents/__init__.py

```diff
@@ -2,10 +2,14 @@
 from . import BasicAgent
 from . import DQRAgent
 from . import RandomAgent
+from . import BasicAgent2
+from . import CEMAgent
 
 __all__ = [
     'TestAgent',
     'BasicAgent',
     'DQRAgent',
-    'RandomAgent'
+    'RandomAgent',
+    'BasicAgent2',
```

Comment:
You didn't include `BasicAgent2` in your diff, but it's required otherwise you can't run the cli

Reply:
I just updated the file and removed BasicAgent2 Thanks!